test/e2e/scrapeconfig_test.go

```diff
@@ -216,7 +216,7 @@ func testPromOperatorStartsWithoutScrapeConfigCRD(t *testing.T) {
 
 	restarts, err := framework.GetPodRestartCount(context.Background(), ns, pl.Items[0].GetName())
 	require.NoError(t, err)
-	require.Emptyf(t, restarts, "expected to get 1 container but got %d", len(restarts))
+	require.Lenf(t, restarts, 1, "expected to get 1 container but got %d", len(restarts))
```

Comment:
Actually we're counting the amount of restarts here, not the amount of containers. The error message is certainly misleading!

That said, the test is correct. We don't expect restarts here